services/button.c

```diff
@@ -58,9 +59,6 @@ void btn_process(srv_t *state) {
 }
 
 void btn_handle_packet(srv_t *state, jd_packet_t *pkt) {
-    if (sensor_handle_packet_simple(state, pkt, &state->pressed, sizeof(state->pressed)))
```

Comment:
you still need that - this handles the case when the client asks for streaming

Reply:
oh, my bad. thanks.